src/sentry/api/serializers/models/project.py

```diff
@@ -306,6 +320,7 @@ def get_attrs(self, item_list, user):
         for item in item_list:
             attrs[item]['latest_release'] = latest_releases.get(item.id)
             attrs[item]['deploys'] = deploys_by_project.get(item.id)
+            attrs[item]['environments'] = environments_by_project.get(item.id)
```

Comment:
can we do `environments_by_project.get(item.id, [])` so this is always a list?

Reply:
Of course. :)